Tests/SentryTests/SentryTransactionTests.swift

```diff
@@ -110,4 +110,34 @@ class SentryTransactionTest: XCTestCase {
         XCTAssertNotNil((serialization["contexts"] as! Dictionary)["trace"])
         XCTAssertNotNil(serialization["spans"])
     }
+    
+    func testAdditionOfChild() {
+        let transaction = Transaction(name: someTransactionName)
+        transaction.startChild(operation: someOperation)
+        XCTAssertEqual(transaction.spans.count, 1)
```

Comment:
`m`: Can we please assert the contents of the span so we make sure it is properly initialized. This applies to the other test methods as well.